collections/nemo_nlp/nemo_nlp/data/datasets/utils.py

```diff
@@ -1021,6 +1024,66 @@ def __init__(self,
         self.num_intents = len(get_vocab(self.intent_dict_file))
         slots = label2idx(self.slot_dict_file)
         self.num_slots = len(slots)
+
+        for mode in ['train', 'test']:
+
+            slot_file = f'{self.data_dir}/{mode}_slots.tsv'
+            with open(slot_file, 'r') as f:
+                slot_lines = f.readlines()
+
+            input_file = f'{self.data_dir}/{mode}.tsv'
+            with open(input_file, 'r') as f:
+                input_lines = f.readlines()[1:]
```

Comment:
future work: we might want to have an option to specify how many first lines of the file to skip instead of just picking 1